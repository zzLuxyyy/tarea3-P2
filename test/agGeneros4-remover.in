crearAGGeneros
insertarAGGeneros -1 100 Historica
insertarAGGeneros 100 101 Romance
insertarAGGeneros 100 102 Terror
insertarAGGeneros 100 103 Historica
insertarAGGeneros 100 104 Terror
imprimirAGGeneros
removerGeneroAGGeneros 104
imprimirAGGeneros
removerGeneroAGGeneros 101
imprimirAGGeneros
insertarAGGeneros 102 202 Romance
insertarAGGeneros 102 206 Historica
insertarAGGeneros 103 250 Romance
insertarAGGeneros 103 260 Terror
insertarAGGeneros 250 251 Historica
insertarAGGeneros 250 252 Terror
imprimirAGGeneros
removerGeneroAGGeneros 102
imprimirAGGeneros
removerGeneroAGGeneros 103
imprimirAGGeneros
liberarAGGeneros
Fin