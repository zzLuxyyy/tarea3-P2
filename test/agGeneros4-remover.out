1>Se ejecutó 'crearTAGGeneros' exitosamente.
2>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre -1 e idGenero 100.
3>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 100 e idGenero 101.
4>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 100 e idGenero 102.
5>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 100 e idGenero 103.
6>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 100 e idGenero 104.
7>Árbol de géneros:
100 - Historica
    104 - Terror
    103 - Historica
    102 - Terror
    101 - Romance
8>El género de id 104 fue removido.
9>Árbol de géneros:
100 - Historica
    103 - Historica
    102 - Terror
    101 - Romance
10>El género de id 101 fue removido.
11>Árbol de géneros:
100 - Historica
    103 - Historica
    102 - Terror
12>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 102 e idGenero 202.
13>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 102 e idGenero 206.
14>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 103 e idGenero 250.
15>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 103 e idGenero 260.
16>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 250 e idGenero 251.
17>Se ejecutó 'insertarGeneroTAGGeneros' con idGeneroPadre 250 e idGenero 252.
18>Árbol de géneros:
100 - Historica
    103 - Historica
        260 - Terror
        250 - Romance
            252 - Terror
            251 - Historica
    102 - Terror
        206 - Historica
        202 - Romance
19>El género de id 102 fue removido.
20>Árbol de géneros:
100 - Historica
    103 - Historica
        260 - Terror
        250 - Romance
            252 - Terror
            251 - Historica
21>El género de id 103 fue removido.
22>Árbol de géneros:
100 - Historica
23>Se ejecutó 'liberarTAGGeneros' exitosamente.
24>Fin.
